fmriprep/utils/bids.py

```diff
@@ -166,5 +168,19 @@ def collect_data(dataset, participant_label, task=None):
     if task:
         queries['bold']['task'] = task
 
-    return {modality: [x.filename for x in layout.get(**query)]
-            for modality, query in queries.items()}, layout
+    subj_data = {modality: [x.filename for x in layout.get(**query)]
+                 for modality, query in queries.items()}
+
+    def _run_num(x):
+        return re.search("run-\\d*", x).group(0)
+
+    if subj_data["bold"] is not []:
```

Comment:
@chrisfilo could you check this bit of code? I don't have a clear understanding of how ME works in BIDS. Thanks!